recipe-box-backend/src/main/java/com/endava/recipebox/dto/RecipeRequestDTO.java

```diff
@@ -0,0 +1,23 @@
+package com.endava.recipebox.dto;
+
+import com.endava.recipebox.model.Difficulty;
+import com.endava.recipebox.model.MealType;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.Duration;
+import java.util.List;
+
+@Getter
+@Setter
+public class RecipeRequestDTO {
```

Comment:
because is the DTO used for getting the request body for adding a recipe, a more intuitive naming is RecipeAddRequestDTO, in this way you will know from a glance that the dto is used as request body for POST method, without the need to search for the usage of DTO.